### DISCUSSION TITLE
How to use preflight styles in non-TailwindCSS projects

### DISCUSSION THREAD

**Discussion by mrcaidev** [tailwindlabs/tailwindcss#10722] [Help]
Tailwind's preflight styles are awesome, and I want to use it in projects with native CSS. Is there an elegant way to do this?

It's ok to simply copy and paste the file, but I want to keep it always up to date.

Maybe it would be a good idea to publish a seperate package for the preflight styles?

**Comment by Gravy59** [comment]
You can use [`modern-normalize`](https://github.com/sindresorhus/modern-normalize). Then, add a new CSS file and do this:
- add a solid 0-width gray border to `*, ::before, ::after`
- inherit line height on `body`
- set the border-top-width of `hr` to 1px
- copy styles from [Tailwind Preflight](https://tailwindcss.com/docs/preflight)

This actually makes me want to make a CSS file that merges the two

**Reply by Gravy59** [reply]
Done, [here it is](https://github.com/gravy59/fullreset)

```bash
npm i -D fullreset
```

**Reply by mrcaidev** [reply]
Brilliant work! But what's the point of step 1 2 3 when you directly copy the styles from Tailwind in step 4

**Reply by Gravy59** [reply]
Tailwind Preflight uses Postcss to compile theme variables, so just make sure to remove them. Or just use fullreset.
